csl-data.json

```diff
@@ -1,431 +1,440 @@
-                    "$ref": "name-variable" 
-                } 
-            },
-            "editorial-director": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "interviewer": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "illustrator": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "original-author": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "recipient": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "reviewed-author": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "translator": {
-                "type": "array",
-                "items": {
-                    "$ref": "name-variable" 
-                } 
-            },
-            "accessed": {
-                "id": "date-variable",
-                "type": [
+                    ]
+                },
+                {
+                    "properties": {
+                        "circa": {
+                            "type": [
+                                "string",
+                                "number",
+                                "boolean"
+                            ]
+                        },
+                        "season": {
+                            "type": [
+                                "number"
+                            ],
+                            "minimum": 1,
+                            "maximum": 4
+                        }
+                    }
+                }
+            ]
+        },
+        "nameDefinition": {
+            "type": "array",
+            "items": {
+                "anyOf": [
                     {
+                        "type": "object",
                         "properties": {
-                            "date-parts": {
-                                "type": "array",
-                                "items": {
-                                    "type": "array",
-                                    "items": {
-                                        "type": [
-                                            "string",
-                                            "number" 
-                                        ] 
-                                    },
-                                    "maxItems": 3 
-                                },
-                                "maxItems": 2 
+                            "literal": {
```

Comment:
Made this an "either-or" between `literal` and the rest of the fields.

If no `literal`, then it should at least have `family` (see below)